modules/fb_instant_articles_display/fb_instant_articles_display.module

```diff
@@ -13,15 +13,15 @@
 */
 function fb_instant_articles_display_init() {
  // Configure Instant Articles Transformer logger to append to Drupal log if enabled
- $enable_transformer_logging = variable_get('fb_instant_articles_display_enable_transformer_logging', FALSE);
+ $enable_transformer_logging = variable_get('fb_instant_articles_display_enable_transformer_logging');
```

Comment:
Good point regarding the chatti-ness of the transformer.  It also logs lines of the same message with separate calls.  From what I've read, I'm not sure thats the intended use.  At least, it comes out kind of awkward in the Drupal DBLog module the way we have it setup, eg.

http://take.ms/y59qK

Since the Drupal watchdog displays the most recent log message first, you have to read the set of message from a single set, if you will, backwards.

We should probably do something about this, although, total post-F8 task.  Wondering if this should get fixed upstream in the SDK?

I'll volunteer to write up the issue in the right spot.